AndroidSDK/src/com/leanplum/Leanplum.java

```diff
@@ -628,6 +629,9 @@ private static void startHelper(
     Date now = new Date();
     int timezoneOffsetSeconds = localTimeZone.getOffset(now.getTime()) / 1000;
 
+    String registrationId = SharedPreferencesUtil.getString(context,
```

Comment:
this not needed, before send request we will attach appId etc: https://github.com/Leanplum/Leanplum-Android-SDK/blob/develop/AndroidSDK/src/com/leanplum/internal/Request.java#L293

plus when you createArgsDictionary  - you added deviceID and userID
https://github.com/Leanplum/Leanplum-Android-SDK/blob/develop/AndroidSDK/src/com/leanplum/internal/Request.java#L185